limesurvey/limesurvey.py

```diff
@@ -244,6 +292,12 @@ def add_participant_to_survey(self, user, anonymous_user_id: str):
             user: The user to add as participant
             anonymous_user_id: The anonymous user ID of the user
         """
+        try:
+            self.check_user_in_survey(anonymous_user_id)
+            return None
+        except NoParticipantFound:
+            pass
```

Comment:
Why use a `pass` here?

Reply:
If the participant is not found, then add it. If found, then return. We need to continue -- to add the student. if the exception is raised